gae/js/app/views/staffposition_rates.js

```diff
@@ -0,0 +1,132 @@
+define(
+    [
+        'backbone', 'backform', 'bootstrap','bootstrap-datepicker',
+        'text!app/templates/staffposition_rates.html', 'app/models/rate_after_date',
+    ],
+    function(Backbone, Backform, bootstrap, bsdp, template, RateAfterDate) {
+        var fields = [
+            {
+                name: "rate",
+                control: "input",
+                updateLabel: function(name){
+                    this.label = name + ' Rate';
+                }
+            },{
+                name: "date",
+                control: "datepicker",
+                options: {autoclose: true, assumeNearbyYear: true, todayHighlight: true, format: "yyyy-mm-dd", orientation: 'bottom right'},
+                updateLabel: function(name){
+                    this.label = name + ' Date';
+                }
+            },{
+                control: "button",
+                extraClasses: ['btn-primary', 'btn-sm'],
+                name: "updateRates",
+                updateBtn: function(name, action){
+                    this.label = action + name + ' Rate';
+                }
+            },
+        ];
+        var View = Backbone.View.extend({
+            el: '#simple-form-view',
+            events: {
+                'click th.staffposition-btn.new-rad': 'newRate',
+                'click td .staffposition-btn.edit': 'editRate',
+                'click .staffposition-btn.remove': 'removeRate',
+                'click button[name=updateRates]': 'updateRates',
+                'click button[name=cancel]': 'render',
+            },
+            initialize: function(options) {
+                var self = this;
+                this.options = options;
+                this.name = options.name;
+                this.templateName = options.templateName;
+                this.staffposition = options.staffposition;
+                this.loading = options.loading;
+                this.rates = this.staffposition.get(this.name);
+                this.template = _.template(template);
+                this.views = options.views;
+                $.fn.modal.Constructor.Default.backdrop = false
+                this.modalId = "#".concat(this.name).concat('Modal');
+
+                this.listenTo(this.staffposition, 'change',
```

Comment:
This can listen for change on the collection and re-render.  We don't need to worry about loading here because it's not a view that is the root of a page load.